components/outlier-detection/isolation-forest/isolation_forest.ipynb

```diff
@@ -271,16 +276,23 @@
    },
    "outputs": [],
    "source": [
-    "!helm install ../../../helm-charts/seldon-core --name seldon-core \\\n",
-    "        --namespace seldon \\\n",
-    "        --set ambassador.enabled=true"
+    "!kubectl rollout status statefulset.apps/seldon-operator-controller-manager -n seldon-system"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Check deployment rollout status for seldon core."
+    "Install Ambassador API gateway"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!helm install stable/ambassador --name ambassador --set image.tag=0.40.2"
```

Comment:
When testing I had issues with using this version ambassador.

Version "0.40.2" failed to start on the gke cluster I was using to test, however the following versions of ambassador worked fine

"0.53.1" <- the version of ambassador in seldon-core helm dependency at time of testing
"0.61.0" <- the latest version of ambassador at time of testing

Reply:
There still seem to be issues with Ambassador 0.61.0 with gRPC so I think its best we remain suggesting 0.40.2 until these are clarified. https://github.com/datawire/ambassador/issues/504